DescendantBooks/DetailedDescendantBookReport.py

```diff
@@ -442,11 +444,11 @@ def write_report(self):
             self.report_count += 1
             if len(self.ascendants) > 1:
                 self.title = \
-                    self._("%(report_count)s. Descendant Report for " \
+                    _("%(report_count)s. Descendant Report for " \
```

Comment:
This will break the deferred translation functionality.

Reply:
Do you mean that I must revert all "\_(" to "self.\_(" or only here?

Reply:
Everywhere.  It's needed to run reports in different languages.